### ISSUE TITLE
generalise this line.

### ISSUE THREAD

**Issue by julesghub** [jmansour/scaling_scripts#1]
https://github.com/jmansour/scaling_scripts/blob/dcdb83c9897056d0193e1d6e5fb0d92958c18c7a/rt_timed.py#L213

I'm just wondering how critical this is? I was thinking of removing all raijin / pawsey / environment specific code and putting it in the script that invokes qsub or sbatch. Does that seem right to you @jmansour ?
 I'm happy to do it. Just wanted to run it by you.

**Comment by jmansour** [comment]
Yeah certainly more than happy for it to be generalised. I'm the case of
the line above, as long as it writes out a unique job I'd, that's fine. The
Id is necessary as it allows us to run identical jobs numerous times so
that we can average over the results.
